# AI-for-Pathology
## Surveys
+ Graph Neural Networks in Histopathology: Emerging Trends and Future Directions, arXiv 2024. [[Paper]](https://arxiv.org/pdf/2406.12808)
+ Foundation model for advancing healthcare: Challenges, opportunities, and future directions, arXiv 2024. [[Paper]](https://arxiv.org/pdf/2404.03264) [[Project]](https://github.com/YutingHe-list/Awesome-Foundation-Models-for-Advancing-Healthcare)
+ A Survey of Large Language Models for Healthcare: from Data, Technology, and Applications to Accountability and Ethics, arXiv 2024. [[Paper]](https://arxiv.org/pdf/2310.05694) [[Project]](https://github.com/KaiHe-better/LLM-for-Healthcare)
+ Data-Centric Foundation Models in Computational Healthcare: A Survey, arXiv 2024. [[Paper]](https://arxiv.org/pdf/2401.02458)
+ Large language models and multimodal foundation models for precision oncology, NPJ Precision Oncology 2024. [[Paper]](https://www.nature.com/articles/s41698-024-00573-2)
+ Artificial intelligence for digital and computational pathology, Nature Reviews Bioengineering 2023. [[Paper]](https://arxiv.org/pdf/2401.06148)
+ Foundational models in medical imaging: A comprehensive survey and future vision, arXiv 2023. [[Paper]](https://arxiv.org/pdf/2310.18689) [[Project]](https://github.com/xmindflow/Awesome-Foundation-Models-in-Medical-Imaging)
+ Deep learning-enabled virtual histological staining of biological samples, Light: Science & Applications, 2023. [[Paper]](https://www.nature.com/articles/s41377-023-01104-7.pdf)
## Papers
### VLM
+ Multimodal Whole Slide Foundation Model for Pathology, arXiv 2024. [[Paper]](https://arxiv.org/pdf/2411.19666) [[Code]](https://github.com/mahmoodlab/TITAN)
+ Towards A Generalizable Pathology Foundation Model via Unified Knowledge Distillation, arXiv 2024. [[Paper]](https://arxiv.org/pdf/2407.18449)
+ MambaMIL: Enhancing Long Sequence Modeling with Sequence Reordering in Computational Pathology, ECCV 2024. [[Paper]](https://arxiv.org/pdf/2403.06800) [[Code]](https://arxiv.org/pdf/2403.06800)
+ PathoTune: Adapting Visual Foundation Model to Pathological Specialists, MICCAI 2024. [[Paper]](https://arxiv.org/pdf/2403.16497) [[Code]](https://github.com/openmedlab/PathoDuet)
+ Controllable and Efficient Multi-Class Pathology Nuclei Data Augmentation using Text-Conditioned Diffusion Models, MICCAI 2024. [[Paper]](https://arxiv.org/pdf/2407.14426)
+ Towards a text-based quantitative and explainable histopathology image analysis, MICCAI 2024. [[Paper]](https://arxiv.org/pdf/2407.07360) [[Code]](https://github.com/QuIIL/TQx)
+ CausalCellSegmenter: Causal Inference inspired Diversified Aggregation Convolution for Pathology Image Segmentation, MICCAI 2024. [[Paper]](https://arxiv.org/pdf/2403.06066)
+ Reducing self-supervised learning complexity improves weakly-supervised classification performance in computational pathology, MICCAI 2024. [[Paper]](https://arxiv.org/pdf/2403.04558)
+ Prompting Vision Foundation Models for Pathology Image Analysis, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Yin_Prompting_Vision_Foundation_Models_for_Pathology_Image_Analysis_CVPR_2024_paper.pdf) [[Code]](https://github.com/7LFB/QAP)
+ Feature Re-Embedding: Towards Foundation Model-Level Performance in Computational Pathology, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Tang_Feature_Re-Embedding_Towards_Foundation_Model-Level_Performance_in_Computational_Pathology_CVPR_2024_paper.pdf) [[Code]](https://github.com/DearCaat/RRT-MIL)
+ Morphological Prototyping for Unsupervised Slide Representation Learning in Computational Pathology, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Song_Morphological_Prototyping_for_Unsupervised_Slide_Representation_Learning_in_Computational_Pathology_CVPR_2024_paper.pdf) [[Code]](https://github.com/mahmoodlab/Panther)
+ Transcriptomics-guided Slide Representation Learning in Computational Pathology, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Jaume_Transcriptomics-guided_Slide_Representation_Learning_in_Computational_Pathology_CVPR_2024_paper.pdf) [[Code]](https://github.com/mahmoodlab/TANGLE)
+ PrPSeg: Universal Proposition Learning for Panoramic Renal Pathology Segmentation, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Deng_PrPSeg_Universal_Proposition_Learning_for_Panoramic_Renal_Pathology_Segmentation_CVPR_2024_paper.pdf)
+  XFibrosis: Explicit Vessel-Fiber Modeling for Fibrosis Staging from Liver Pathology Images, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Yin_XFibrosis_Explicit_Vessel-Fiber_Modeling_for_Fibrosis_Staging_from_Liver_Pathology_CVPR_2024_paper.pdf)
+  Rotation-Agnostic Image Representation Learning for Digital Pathology, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Alfasly_Rotation-Agnostic_Image_Representation_Learning_for_Digital_Pathology_CVPR_2024_paper.pdf) [[Code]](https://kimialabmayo.github.io/PathDino-Page/)
+ Towards a general-purpose foundation model for computational pathology, Nature Medicine 2024. [[Paper]](https://www.nature.com/articles/s41591-024-02857-3.epdf?sharing_token=CzM2TCW_6hilYJ6BCMgx5dRgN0jAjWel9jnR3ZoTv0PwDLGHgijc_t52lQyxVvw552KDCUhMbS4KuO_rvLnP6S1BpmIj9puojkF8lfR5R8uEX08B0FxePgIH0t7DovKvZF4NHQKlq4TZHGAA1wEIdkYKvcr8nUsaa-nNYbNw3JI%3D) [[Code]](https://github.com/mahmoodlab/UNI)
+ Virchow: A Million-Slide Digital Pathology Foundation Model, arXiv 2023. [[Paper]](https://arxiv.org/pdf/2309.07778)
+ Robust and data-efficient generalization of self-supervised machine learning for diagnostic imaging,  nature biomedical engineering 2023. [[Paper]](https://www.nature.com/articles/s41551-023-01049-7)
+ Transformer-based unsupervised contrastive learning for histopathological image classification, Medical Image Analysis 2022. [[Paper]](https://helios2.mi.parisdescartes.fr/~lomn/Cours/ACP/Complement/cTransPAth.pdf) [[Code]](https://github.com/Xiyue-Wang/TransPath)
+ Self supervised contrastive learning for digital histopathology, Machine Learning with Applications 2022. [[Paper]](https://www.sciencedirect.com/science/article/pii/S2666827021000992) [[Code]](https://github.com/ozanciga/self-supervised-histopathology?tab=readme-ov-file)
### LLM
+ BiMediX: Bilingual Medical Mixture of Experts LLM, arXiv 2024. [[Paper]](https://arxiv.org/pdf/2402.13253) [[Code]](https://github.com/mbzuai-oryx/BiMediX)
+ Adapted Large Language Models Can Outperform Medical Experts in Clinical Text Summarization, Nature Medicine 2024. [[Paper]](https://arxiv.org/pdf/2309.07430)
+ Towards expert-level medical question answering with large language models, arXiv 2023. [[Paper]](https://arxiv.org/pdf/2305.09617)
+ Large Language Models Encode Clinical Knowledge, Nature 2023. [[Paper]](https://arxiv.org/pdf/2212.13138)
+ Huatuogpt-ii, one-stage training for medical adaption of llms, COLM 2024. [[Paper]](https://arxiv.org/pdf/2311.09774) [[Code]](https://github.com/FreedomIntelligence/HuatuoGPT-II)
+ HuatuoGPT, towards Taming Language Model to Be a Doctor, EMNLP 2023. [[Paper]](https://arxiv.org/pdf/2305.15075) [[Code]](https://github.com/FreedomIntelligence/HuatuoGPT)
### MLM
+ A Multimodal Knowledge-enhanced Whole-slide Pathology Foundation Model, arXiv 2024. [[Paper]](https://arxiv.org/abs/2407.15362)
+ Decomposing Disease Descriptions for Enhanced Pathology Detection: A Multi-Aspect Vision-Language Pre-training Framework, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Phan_Decomposing_Disease_Descriptions_for_Enhanced_Pathology_Detection_A_Multi-Aspect_Vision-Language_CVPR_2024_paper.pdf) [[Code]](https://github.com/HieuPhan33/CVPR2024_MAVL)
+ ViLa-MIL: Dual-scale Vision-Language Multiple Instance Learning for Whole Slide Image Classification, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Shi_ViLa-MIL_Dual-scale_Vision-Language_Multiple_Instance_Learning_for_Whole_Slide_Image_CVPR_2024_paper.pdf) [[Code]](https://github.com/Jiangbo-Shi/ViLa-MIL)
+ A Multimodal Generative AI Copilot for Human Pathology, Nature 2024. [[Paper]](https://www.nature.com/articles/s41586-024-07618-3) [[AiXiv]](https://arxiv.org/pdf/2312.07814) [[Code]](https://github.com/fedshyvana/pathology_mllm_training)
+ A visual-language foundation model for computational pathology, Nature Medicine 2024. [[Paper]](https://www.nature.com/articles/s41591-024-02856-4.epdf?sharing_token=oxrnoCI4J4GotgQPtYqhxtRgN0jAjWel9jnR3ZoTv0PmeocnzHdxO4Q-z304lTFOE9gfAQ4aOUdEe1q7KpOZrJPvH2xnU6NL0L0gf1Wy5LPTCE_ptjKfk7dLA2eG5opNjEHop5oOAfB60GZRNBQbXXwltaWADikm_fOMbypyiWA%3D) [[Code]](https://github.com/mahmoodlab/CONCH)
+ A visualâ€“language foundation model for pathology image analysis using medical Twitter, Nature Medicine 2024. [[Paper]](https://www.nature.com/articles/s41591-023-02504-3.epdf?sharing_token=2umlCrKLgEIF8vmuLpQ7AtRgN0jAjWel9jnR3ZoTv0NWSxjlTuWM3jUBxiqED7ai3ueIDYQ_xX2BBBGXn0IDY_RMdGid_ppbXRxR40prhjrWvtzO3o_QB1gW6NTYt8EB0UO5VjWecg4rWh3LM_L-Rf59L6s9Fx7yR521Lp3GfhU%3D) [[Code]](https://huggingface.co/spaces/vinid/webplip)
+ Pathasst: Redefining pathology through generative foundation ai assistant for pathology, AAAI 2024. [[Paper]](https://arxiv.org/pdf/2305.15072) [[Code]](https://github.com/superjamessyx/Generative-Foundation-AI-Assistant-for-Pathology)
+ Quilt-1m: One million image-text pairs for histopathology, NeurIPS 2023. [[Paper]](https://proceedings.neurips.cc/paper_files/paper/2023/file/775ec578876fa6812c062644964b9870-Paper-Datasets_and_Benchmarks.pdf) [[Code]](https://quilt1m.github.io/)
+ Visual Language Pretrained Multiple Instance Zero-Shot Transfer for Histopathology Images, CVPR 2023. [[Paper]](https://openaccess.thecvf.com/content/CVPR2023/papers/Lu_Visual_Language_Pretrained_Multiple_Instance_Zero-Shot_Transfer_for_Histopathology_Images_CVPR_2023_paper.pdf) [[Code]](https://github.com/mahmoodlab/MI-Zero)
+ Text-guided foundation model adaptation for pathological image classification, MICCAI 2023. [[Paper]](https://arxiv.org/pdf/2307.14901) [[Code]](https://github.com/Yunkun-Zhang/CITE)
### T2I Generation 
+ Co-synthesis of Histopathology Nuclei Image-Label Pairs using a Context-Conditioned Joint Diffusion Model, ECCV 2024. [[Paper]](https://arxiv.org/pdf/2407.14434)
+ Controllable and Efficient Multi-Class Pathology Nuclei Data Augmentation using Text-Conditioned Diffusion Models, MICCAI 2024. [[Paper]](https://arxiv.org/pdf/2407.14426)
+ Learned representation-guided diffusion models for large-image generation, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Graikos_Learned_Representation-Guided_Diffusion_Models_for_Large-Image_Generation_CVPR_2024_paper.pdf) [[Code]](https://github.com/cvlab-stonybrook/Large-Image-Diffusion?tab=readme-ov-file)
+ GenSelfDiff-HIS: Generative Self-Supervision Using Diffusion for Histopathological Image Segmentation, ArXiv 2023. [[Paper]](https://arxiv.org/pdf/2309.01487)
+ PathLDM: Text conditioned Latent Diffusion Model for Histopathology, WACV 2024. [[Paper]](https://openaccess.thecvf.com/content/WACV2024/papers/Yellapragada_PathLDM_Text_Conditioned_Latent_Diffusion_Model_for_Histopathology_WACV_2024_paper.pdf) [[Code]](https://github.com/cvlab-stonybrook/PathLDM)
+ ViT-DAE: Transformer-driven Diffusion Autoencoder for Histopathology Image Analysis, MICCAI 2023. [[Paper]](https://arxiv.org/pdf/2304.01053)
### I2I Generation
+ Advancing H&E-to-IHC Virtual Staining with Task-Specific Domain Knowledge for HER2 Scoring, MICCAI 2024. [[Paper]](https://papers.miccai.org/miccai-2024/paper/3227_paper.pdf) [[Code]](https://github.com/balball/TDKstain)
+ Boosting FFPE-to-HE Virtual Staining with Cell Semantics from Pretrained Segmentation Model, MICCAI 2024. [[Paper]](https://papers.miccai.org/miccai-2024/paper/3335_paper.pdf) [[Code]](https://github.com/huyihuang/FFPE-to-HE)
+ Exploiting Supervision Information in Weakly Paired Images for IHC Virtual Staining, MICCAI 2024. [[Paper]](https://papers.miccai.org/miccai-2024/paper/3332_paper.pdf)
+ Pathological Semantics-Preserving Learning for H&E-to-IHC Virtual Staining, MICCAI 2024. [[Paper]](https://papers.miccai.org/miccai-2024/paper/2078_paper.pdf) [[Code]](https://github.com/ccitachi/PSPStain)
+ DiffMix: Diffusion Model-based Data Synthesis for Nuclei Segmentation and Classification in Imbalanced Pathology Image Datasets, MICCAI 2024. [[Paper]](https://arxiv.org/pdf/2306.14132)
+ Virtual Immunohistochemistry Staining for Histological Images Assisted by Weakly-supervised Learning, CVPR 2024. [[Paper]](https://openaccess.thecvf.com/content/CVPR2024/papers/Li_Virtual_Immunohistochemistry_Staining_for_Histological_Images_Assisted_by_Weakly-supervised_Learning_CVPR_2024_paper.pdf) [[Code]](https://github.com/jiahanli2022/confusion-GAN)
+ Virtual histological staining of unlabeled autopsy tissue, Nature Communications 2024. [[Paper]](https://www.nature.com/articles/s41467-024-46077-2.pdf) [[Code]](https://github.com/liyuzhu1998/Autopsy-Virtual-Staining/tree/main)
+ Deep learning-based virtual staining, segmentation, and classification in label-free photoacoustic histology of human specimens, Light: Science & Applications, 2024. [[Paper]](https://www.nature.com/articles/s41377-024-01554-7.pdf) [[Code]](https://github.com/YoonChiHo/DL-based-framework-for-automated-HIA-of-label-free-PAH-images)
+ Accelerating histopathology workflows with generative AI-based virtually multiplexed tumour profiling, Nature Machine Intelligence 2024. [[Paper]](https://www.nature.com/articles/s42256-024-00889-5.pdf) [[Code]](https://github.com/AI4SCR/VirtualMultiplexer)
+ Efficient Supervised Pretraining of Swin-Transformer for Virtual Staining of Microscopy Images, TMI 2024. [[Paper]](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=10328980&casa_token=odssktYO2RwAAAAA:uutFKPeBtzTGdxoGaAZ159cLlYl4U7GSZnnbZZM_I0nGes_ivz6lEkBeWkB9Pc4pZX0yfs_Q4mM) [[Code]](https://github.com/birkhoffkiki/CAS-Transformer)
+ PST-Diff: Achieving High-consistency Stain Transfer by Diffusion Models with Pathological and Structural Constraints, TMI 2024. [[Paper]](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=10601703&casa_token=6ZeQbdZo83kAAAAA:LTuZ3L3cNe9VHcxdRZSoXqCObJu3G22AJFP_aGUX4LG6h_QZvSIjRf6kcBsWyhKkFgdLlLOgIOk)
+ PathoDuet: Foundation models for pathological slide analysis of H&E and IHC stains, Medical Image Analysis 2024. [[Paper]](https://www.sciencedirect.com/science/article/pii/S1361841524002147?casa_token=3T5rOS0PhFIAAAAA:HFrJUB9u6wQJcGtgYuKutzLxnJBi4Ks-iWqiPzaf9Fg3tNFlMcpsdmDrqK1iB8PCXIoJTU2dVwg) [[Code]](https://github.com/openmedlab/PathoDuet)
+ Adaptive Supervised PatchNCE Loss for Learning H&E-to-IHC Stain Translation with Inconsistent Groundtruth Image Pairs, MICCAI 2023. [[Paper]](https://arxiv.org/pdf/2303.06193) [[Code]](https://github.com/lifangda01/AdaptiveSupervisedPatchNCE)
+ StainDiff: Transfer Stain Styles of Histology Images with Denoising Diffusion Probabilistic Models and Self-Ensemble, MICCAI 2023. [[Paper]](https://link.springer.com/epdf/10.1007/978-3-031-43987-2_53?sharing_token=USlvHa4KCa0yfgjwzi-v0_e4RwlQNchNByi7wbcMAY7jxNo0bliUewITgRTD3ZK5MCg3ig12I68XFqBSO8sjxfbXg6aF3-OhSFqLEIV2d97CJYVwsb-jQ6HCOj1p_9QFw0dKRg8Yf_TR7izxU2spWiJV_1MQXpt80NDAsZ4JwLs%3D)
+ MulHiST: Multiple Histological Staining for Thick Biological Samples via Unsupervised Image-to-Image Translation, MICCAI 2023. [[Paper]](https://link.springer.com/epdf/10.1007/978-3-031-43987-2_71sharing_token=vw71r1FBJEDBpzyS6_MUzPe4RwlQNchNByi7wbcMAY7jxNo0bliUewITgRTD3ZK59goddOmnxh-67oF2XQOZKNcmmAnr0vwIpAWBZbwflK7PMvqbJ3MziqUy20lIdFW7cMf8IHIjEPbsBrwBujH2qijEfOvr6xxRNJMJtSqDRrw%3D) [[Code]](https://github.com/TABLAB-HKUST/MulHiST)
+ BCI: Breast Cancer Immunohistochemical Image Generation through Pyramid Pix2pix, CVPRW 2022. [[Paper]](https://openaccess.thecvf.com/content/CVPR2022W/CVMI/papers/Liu_BCI_Breast_Cancer_Immunohistochemical_Image_Generation_Through_Pyramid_Pix2pix_CVPRW_2022_paper.pdf) [[Code]](https://bupt-ai-cz.github.io/BCI/)
### I2R Generation
+ Learning to predict RNA sequence expressions from whole slide images with applications for search and classification,  Communications Biology 2024. [[Paper]](https://www.nature.com/articles/s42003-023-04583-x.pdf)
+ Digital profiling of gene expression from histology images with linearized attention, Nature Communications 2024. [[Paper]](https://www.nature.com/articles/s41467-024-54182-5.pdf) [[Code]](https://github.com/gevaertlab/sequoia-pub)
### Others
+ Hidden flaws behind expert-level accuracy of multimodal GPT-4 vision in m dicine, npj digital medicine 2024. [[Paper]](https://www.nature.com/articles/s41746-024-01185-7?fromPaywallRec=false)
+ Demographic bias in misdiagnosis by computational pathology models, Nature Medicine 2024. [[Paper]](https://www.nature.com/articles/s41591-024-02885-z.epdf?sharing_token=eFiSitgoBRMhuXIq6n8c-tRgN0jAjWel9jnR3ZoTv0NYPqVo1LCdipUMRepKsM_GUUvsvU5K2bZ3A2cu2jGYzRqeePzKrCjvhlJOTbwpdwL8UTZ4JR338f19cDSk1GOHmLQmjtKlpNaQLvJMSABezkCyM0NU06nKhkdOcN4d9TE%3D) [[Code]](https://github.com/mahmoodlab/CPATH_demographics?tab=readme-ov-file)
## Dataset
+ HEST-1k: A Dataset for Spatial Transcriptomics and Histology Image Analysis, NeurIPS 2024. [[Paper]](https://arxiv.org/pdf/2406.16192) [[Project]](https://github.com/mahmoodlab/hest)
+ [TCGA](https://www.cancer.gov/ccg/research/genome-sequencing/tcga)
## People and Groups
+ [Smart Lab](https://hkustsmartlab.github.io/), HKUST.
+ [Mahmood Lab](https://faisal.ai/), Harvard Medical School.
+ [OpenMEDLab](https://github.com/openmedlab)
## Resources
+ [Awesome-Foundation-Models-for-Advancing-Healthcare](https://github.com/YutingHe-list/Awesome-Foundation-Models-for-Advancing-Healthcare)
